018 Passwort-Generator (Längen Auswahl) | F.W.

iobosse03 opened the pull request.

iobosse03 committed:
[018] added an HTML input field for the password length

018 Passwort-Generator (Längen Auswahl) https://app.asana.com/0/1205072348884926/1205158227496893/f

iobosse03 committed:
[018] connected the input field to the function

018 Passwort-Generator (Längen Auswahl) https://app.asana.com/0/1205072348884926/1205158227496893/f

iobosse03 committed:
[018] added a function to check for correct input values

018 Passwort-Generator (Längen Auswahl) https://app.asana.com/0/1205072348884926/1205158227496893/f

SpaTeamAcademy commented:
Funktionen fehlen:
> Im Placeholder des Eingabefeldes werde ich als User über die min/max Länge informiert. Wenn ich versuche außerhalb dieses Bereichs eine Zahl einzugeben, werde ich über einen Informationstext (farbig, unter dem Eingabefeld) darauf hingewiesen.

iobosse03 committed:
[018] changed the placeholder's information

018 Passwort-Generator (Längen Auswahl) https://app.asana.com/0/1205072348884926/1205158227496893/f

iobosse03 committed:
[018] added text to inform the user of invalid input

018 Passwort-Generator (Längen Auswahl) https://app.asana.com/0/1205072348884926/1205158227496893/f

iobosse03 committed:
[018] changed the placeholder to fit better

018 Passwort-Generator (Längen Auswahl) https://app.asana.com/0/1205072348884926/1205158227496893/f

iobosse03 committed:
Merge branch 'main' into 018

# Conflicts:
#	style.css

iobosse03 commented:
Updated branch 018 from main to solve a merge conflict

Code review: F.W.

SpaTeamAcademy commented:
Änderungen im style.css haben dazu geführt, dass die Navbar in diesem Branch nicht mehr gestylet ist. 
Bitte HTML entsprechend anpassen, damit das css wieder greift.

iobosse03 committed:
[018] fixed the navigation bar

018 Passwort-Generator (Längen Auswahl) https://app.asana.com/0/1205072348884926/1205158227496893/f

iobosse03 commented:
Copied the navbar code from index.html
